src/plugins/console/server/lib/proxy_request.test.ts

```diff
@@ -63,14 +78,7 @@ describe(`Console's send request`, () => {
     } as any;
 
     // Don't set a host header this time
-    const result1 = await proxyRequest({
-      agent: null as any,
-      headers: {},
-      method: 'get',
-      payload: null as any,
-      timeout: 30000,
-      uri: new URL('http://noone.nowhere.none'),
-    });
+    const result1 = await sendProxyRequest({});
```

Comment:
[nit] Clarify naming
```suggestion
    const defaultResult = await sendProxyRequest({});
```